opm/simulators/flow/BlackoilModelEbosNldd.hpp

```diff
@@ -172,6 +172,9 @@ class BlackoilModelEbosNldd {
             domain_linsolvers_.emplace_back(model_.ebosSimulator(), loc_param, force_serial);
         }
 
+        const auto& comm = model_.ebosSimulator().vanguard().grid().comm();
+        rank_ = comm.rank();
```

Comment:
Is there a reason for initialising `rank_` in the body of the constructor instead of the in initialiser list?